.gitignore

```diff
@@ -16,3 +16,7 @@ __pycache__/
 .quilt_checked
 .source_dir
 ipkg-*/
+
+# vim
+*.swp
```

Comment:
That's OK, but should be on a separate commit

Reply:
If it is fine, I would prefer to skip the separation for this time only.

Reply:
Actually this change is not ok.
See Arnout's excellent comment here: https://github.com/prplfoundation/prplMesh/pull/308#discussion_r332587456

This is something that has nothing to do with prplMesh, but is only related to the editor you use.
You can configure it locally either for a specific repository or globally.

Reply:
Makes sense.
@RanRegev, please remove this commit.

Reply:
> Actually this change is not ok.
> See Arnout's excellent comment here: [#308 (comment)](https://github.com/prplfoundation/prplMesh/pull/308#discussion_r332587456)
> 
> This is something that has nothing to do with prplMesh, but is only related to the editor you use.
> You can configure it locally either for a specific repository or globally.
Done according to the recommendation in #308 (comment).